### DISCUSSION TITLE
How to type `Hst` prop in Svelte story?

### DISCUSSION THREAD

**Discussion by danawoodman** [histoire-dev/histoire#289] [Q&A] [answered]
I'm using TypeScript in my stories but I don't see how I can type the `Hst` export without giving it the type `any`

```html
<script lang="ts">
	export let Hst: any;
</script>
```

Is there exported types we can use to type the exported prop?

**Comment by Akryum** [comment, marked as answer]
This is WIP and will come soon :)

**Reply by abeldebruijn** [reply]
I would love this feature. What would be an estimated release date?

**Reply by Akryum** [reply]
Implemented in https://github.com/histoire-dev/histoire/commit/287193b69812dbd07ae66f81ea0d04c23572a293
